### ISSUE TITLE
[a11y tool] discoverability

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-RespEq#41]
We need to somehow alert the user that the tool is available.

Maybe role=application helps (though @zorkow had problems with it in the past). Otheriwse, we could add a label. I'm thinking it might be something equivalent to role=math, i.e., something that will make AT indicate there's some math.

**Comment by zorkow** [comment]
See #48

**Comment by zorkow** [comment]
PR #52 adds an initial announcement.

**Comment by zorkow** [comment]
Initial announcement does not work with NVDA. Maybe we are too polite.

**Comment by pkra** [comment]
Closed by #52
